### ISSUE TITLE
Task - Applicant - ARIA issues

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/TalentCloud#5427]
# Description

1. _Focusable descendents within an `[aria-hidden="true"]` element prevent those interactive elements from being available to users of assistive technologies like screen readers._
- Applications Index > Application Deletion: Are you sure you want to delete this application?
- Job Poster > Work Environment and Culture
- Application Timeline Experience > Add Experience

2. _The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies._
- Application Timeline Skills > Justification

3. When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers.
- Application Timeline Fit > Progress Bar

# Images
![Screen Shot 2021-03-12 at 11 33 20](https://user-images.githubusercontent.com/3046459/110971168-8815e980-8328-11eb-9c22-f86f06d2eb51.png)
![Screen Shot 2021-03-12 at 11 57 33](https://user-images.githubusercontent.com/3046459/110972861-7c2b2700-832a-11eb-89a6-182cedc5bf78.png)
![Screen Shot 2021-03-12 at 12 01 14](https://user-images.githubusercontent.com/3046459/110973257-01aed700-832b-11eb-8722-8122255e86d2.png)
![Screen Shot 2021-03-12 at 12 07 05](https://user-images.githubusercontent.com/3046459/110973688-8d286800-832b-11eb-8254-1a8e97d599cc.png)
![Screen Shot 2021-03-12 at 12 10 59](https://user-images.githubusercontent.com/3046459/110974122-2b1c3280-832c-11eb-9ecb-dbd932240944.png)
